packages/kbn-alerts-ui-shared/src/common/constants/rule_form_flag.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const USE_NEW_RULE_FORM_FEATURE_FLAG = false;
```

Comment:
Shouldn't we define this feature flag in Kibana config?

Reply:
Hey Maryam. It is not something we want users to control. We want to release the new rule form. For that reason, maybe we should remove it entirely. @JiaweiWu Do I miss something?

Reply:
Yea it's just a easy way to not have to fix a bunch of tests and allow folks to test this